packages/web/src/apollo/index.tsx

```diff
@@ -114,6 +115,22 @@ const ApolloProviderWithFetchConfig: React.FunctionComponent<{
    */
   const { isAuthenticated, getToken, type: authProviderType } = useAuth()
 
+  // Keeps track of the most recent req/res data so they can be passed into
+  // any errors passed up to a error boundary.
+
+  const data = {
+    mostRecentRequest: undefined,
+    mostRecentResponse: undefined,
+  } as any
+
+  const updateDataApolloLink = new ApolloLink((operation, forward) => {
+    data.mostRecentRequest = operation
+    return forward(operation).map((result) => {
+      data.mostRecentResponse = result
+      return result
+    })
+  })
```

Comment:
_if_ you want the default error messages to include the request/response info, this needs to be added somewhere. This adds a link into the current chain which keeps track of the res/req so that later it can be used when an error boundary is hit.